app/src/main/res/values/settings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="facebook_app_id">2326266780745811</string>
+    <string name="facebook_login_protocol_scheme">fb2326266780745811</string>
+    <string name="reachfive_scheme">reachfive-lscdhbotrpeh8rflz6bb</string>
```

Comment:
Does it only concern the sandbox?
Would it mean the app won't work with the default identity client any more?